frontend/apps/web/components/site-header/AccountStatusHandler.tsx

```diff
@@ -35,3 +45,69 @@ export function AccountStatusHandler(props: Props) {
     </div>
   );
 }
+
+interface TrialCountdownProps {
+  createdDate: number;
+  isAccountStatusValidResp: IsAccountStatusValidResponse | undefined;
+}
+
+const TRIAL_DURATION_DAYS = 14;
+
+// TODO: created_at is coming back as blank
+
+function TrialCountdown(props: TrialCountdownProps) {
+  const { createdDate, isAccountStatusValidResp } = props;
```

Comment:
As a heads up, we don't currently store our created at with a timestamp, so it comes back on the server as the date in UTC (which is fine because we handle all of the billing in UTC).
but you'll want to make sure all of the date handling here is done in UTC format.